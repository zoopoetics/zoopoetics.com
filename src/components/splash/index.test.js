import React from 'react';
import renderer from 'react-test-renderer';
import Splash from './index';

describe('Splash', () => {
  let component;
  let instance;

  beforeEach(() => {
    component = renderer.create(<Splash />);
    instance = component.getInstance();
  });

  it('matches snapshot', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
