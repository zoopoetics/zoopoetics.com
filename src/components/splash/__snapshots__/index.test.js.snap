// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash matches snapshot 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  min-height: 320px;
  min-width: 320px;
  padding-left: 2rem;
  padding-right: 2rem;
  position: absolute;
  width: 100vw;
}

.c1 {
  font-weight: normal;
  margin: 0;
  text-align: center;
}

.c2 {
  font-weight: normal;
  text-align: center;
}

<main
  className="c0"
>
  <h1
    className="c1"
  >
    Peter Barker
  </h1>
  <h2
    className="c2"
  >
    writes code
  </h2>
  <ExternalLinks />
</main>
`;
